src/Clapi/Valuespace.hs

```diff
@@ -257,9 +250,9 @@ validateRoseTree def t = case t of
       fmap (validateWireValues (alValues alTreeTypes) . snd . snd) $
       Map.mapKeys Just $ Dkmap.valueMap m
     _ -> fail "Unexpected time series data!"
-  RtContainer alCont -> case def of
```

Comment:
Was this intentionally warnful?

Reply:
Quite possibly, I'll revert these to remind us to go back and check.